deploy/olm-catalog/knative-serving-operator/0.7.1/knative-serving-operator.v0.7.1.clusterserviceversion.yaml

```diff
@@ -95,9 +95,19 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: knative-serving-operator
+                - name: IMAGE_activator
```

Comment:
This list currently lacks the queue image.

Reply:
Good call. Fixed.